grails-app/controllers/mama/ng/scheduler/MessageController.groovy

```diff
@@ -0,0 +1,110 @@
+package mama.ng.scheduler
+
+import grails.converters.JSON
+
+class MessageController {
+
+    static allowedMethods = [save: "POST", update: "PUT", delete: "DELETE"]
+
+    def cronParserService
```

Comment:
Adds restrictions on what methods are allowed. Remove delete and the controller will no longer accept a delete call.